--- msec-0.80.10/conf/perm.fileserver.rpm_perms~	2012-06-04 22:18:38.760439717 +0200
+++ msec-0.80.10/conf/perm.fileserver	2012-06-04 22:19:35.121735102 +0200
@@ -3,7 +3,7 @@
 /						root.adm		755
 /bin/						root.root		755
 /bin/ping					root.root		4755
-/bin/rpm					rpm.rpm			755
+/bin/rpm					root.root			755
 /boot/						root.root		755
 /dev/						root.root		755
 /etc/						root.root		755
@@ -53,14 +53,14 @@
 /usr/bin/telnet					root.root		755
 /usr/bin/w					root.root		755
 /usr/bin/who					root.root		755
-/usr/lib/rpm/rpm?				rpm.rpm			755
+/usr/lib/root.root?				root.root			755
 /usr/sbin/					root.root		755
 /usr/sbin/sendmail.postfix			root.root		755
 /usr/sbin/sendmail.sendmail			root.mail		2755
 /usr/sbin/traceroute				root.bin		4755
 /usr/tmp					root.root		1777
 /var/						root.root		755
-/var/lib/rpm/Packages				rpm.rpm			644
+/var/lib/rpm/Packages				root.root			644
 /var/lock/subsys				root.root		755
 /var/log/					root.root		755
 /var/log/security.log				root.adm		640 force
--- msec-0.80.10/conf/perm.netbook.rpm_perms~	2012-06-04 22:18:45.066360881 +0200
+++ msec-0.80.10/conf/perm.netbook	2012-06-04 22:19:35.121735102 +0200
@@ -3,7 +3,7 @@
 /						root.adm		755
 /bin/						root.root		755
 /bin/ping					root.root		4755
-/bin/rpm					rpm.rpm			755
+/bin/rpm					root.root			755
 /boot/						root.root		755
 /dev/						root.root		755
 /etc/						root.root		755
@@ -54,14 +54,14 @@
 /usr/bin/telnet					root.root		755
 /usr/bin/w					root.root		755
 /usr/bin/who					root.root		755
-/usr/lib/rpm/rpm?				rpm.rpm			755
+/usr/lib/root.root?				root.root			755
 /usr/sbin/					root.root		755
 /usr/sbin/sendmail.postfix			root.root		755
 /usr/sbin/sendmail.sendmail			root.mail		2755
 /usr/sbin/traceroute				root.bin		4755
 /usr/tmp					root.root		1777
 /var/						root.root		755
-/var/lib/rpm/Packages				rpm.rpm			644
+/var/lib/rpm/Packages				root.root			644
 /var/lock/subsys				root.root		755
 /var/log/					root.root		755
 /var/log/security.log				root.adm		640 force
--- msec-0.80.10/conf/perm.secure.rpm_perms~	2012-06-04 22:18:52.217271482 +0200
+++ msec-0.80.10/conf/perm.secure	2012-06-04 22:19:35.122735090 +0200
@@ -3,7 +3,7 @@
 /						root.adm		755
 /bin/						root.adm		755
 /bin/ping					root.ntools		4750
-/bin/rpm					rpm.rpm			750
+/bin/rpm					root.root			750
 /boot/						root.ctools		710
 /dev/						root.root		755
 /etc/						root.adm		755
@@ -71,7 +71,7 @@
 /usr/bin/telnet					root.ntools		750
 /usr/bin/w					root.ntools		750
 /usr/bin/who					root.ntools		750
-/usr/lib/rpm/rpm?				rpm.rpm			750
+/usr/lib/root.root?				root.root			750
 /usr/sbin/					root.adm		751
 /usr/sbin/sendmail.postfix			root.root		711
 /usr/sbin/sendmail.sendmail			root.mail		2711
@@ -80,7 +80,7 @@
 /usr/share/man					root.root		755
 /usr/tmp					root.adm		1773
 /var/						root.root		755
-/var/lib/rpm/Packages				rpm.rpm			640
+/var/lib/rpm/Packages				root.root			640
 #/var/lib/svn					svn.svn			770
 /var/lock/subsys				root.adm		750
 /var/log/					root.adm		751
--- msec-0.80.10/conf/perm.standard.rpm_perms~	2012-06-04 22:18:57.105210375 +0200
+++ msec-0.80.10/conf/perm.standard	2012-06-04 22:19:35.123735078 +0200
@@ -3,7 +3,7 @@
 /						root.adm		755
 /bin/						root.root		755
 /bin/ping					root.root		4755
-/bin/rpm					rpm.rpm			755
+/bin/rpm					root.root			755
 /boot/						root.root		755
 /dev/						root.root		755
 /etc/						root.root		755
@@ -54,14 +54,14 @@
 /usr/bin/telnet					root.root		755
 /usr/bin/w					root.root		755
 /usr/bin/who					root.root		755
-/usr/lib/rpm/rpm?				rpm.rpm			755
+/usr/lib/root.root?				root.root			755
 /usr/sbin/					root.root		755
 /usr/sbin/sendmail.postfix			root.root		755
 /usr/sbin/sendmail.sendmail			root.mail		2755
 /usr/sbin/traceroute				root.bin		4755
 /usr/tmp					root.root		1777
 /var/						root.root		755
-/var/lib/rpm/Packages				rpm.rpm			644
+/var/lib/rpm/Packages				root.root			644
 /var/lock/subsys				root.root		755
 /var/log/					root.root		755
 /var/log/security.log				root.adm		640 force
--- msec-0.80.10/conf/perm.webserver.rpm_perms~	2012-06-04 22:19:02.058148454 +0200
+++ msec-0.80.10/conf/perm.webserver	2012-06-04 22:19:35.124735066 +0200
@@ -3,7 +3,7 @@
 /						root.adm		755
 /bin/						root.root		755
 /bin/ping					root.root		4755
-/bin/rpm					rpm.rpm			755
+/bin/rpm					root.root			755
 /boot/						root.root		755
 /dev/						root.root		755
 /etc/						root.root		755
@@ -54,14 +54,14 @@
 /usr/bin/telnet					root.root		755
 /usr/bin/w					root.root		755
 /usr/bin/who					root.root		755
-/usr/lib/rpm/rpm?				rpm.rpm			755
+/usr/lib/root.root?				root.root			755
 /usr/sbin/					root.root		755
 /usr/sbin/sendmail.postfix			root.root		755
 /usr/sbin/sendmail.sendmail			root.mail		2755
 /usr/sbin/traceroute				root.bin		4755
 /usr/tmp					root.root		1777
 /var/						root.root		755
-/var/lib/rpm/Packages				rpm.rpm			644
+/var/lib/rpm/Packages				root.root			644
 /var/lock/subsys				root.root		755
 /var/log/					root.root		755
 /var/log/security.log				root.adm		640 force
