--- msec-0.80.10/cron-sh/scripts/03_rpm.sh~	2012-06-04 22:04:17.931201562 +0200
+++ msec-0.80.10/cron-sh/scripts/03_rpm.sh	2012-06-04 22:04:46.825840329 +0200
@@ -39,7 +39,7 @@ fi
 
 # list of installed packages
 if check_is_enabled "${CHECK_RPM_PACKAGES}" ; then
-    rpm -qa --qf "%{NAME}-%{VERSION}-%{RELEASE}\n" | sort > ${RPM_QA_TODAY}
+    rpm -qa | sort > ${RPM_QA_TODAY}
     Filter ${RPM_QA_TODAY} CHECK_RPM_PACKAGES
     Count ${INFOS} ${RPM_QA_TODAY} "Total of installed packages"
     Diffcheck ${RPM_QA_TODAY} ${RPM_QA_YESTERDAY} ${RPM_QA_DIFF} "packages"
